src/lib/components/wizard.component.spec.ts

```diff
@@ -96,16 +87,16 @@ describe('WizardComponent', () => {
       'horizontal': true, 'vertical': false, 'small': true,
       'large-filled': false, 'large-filled-symbols': false, 'large-empty': false, 'large-empty-symbols': false
     });
-    expect(wizard.classes).toEqual({'horizontal': true, 'vertical': false});
+    expect(wizardEl.classes).toEqual({'horizontal': true, 'vertical': false});
     expect(wizardStepsDiv.classes).toEqual({'wizard-steps': true, 'horizontal': true, 'vertical': false});
   });
 
   it('should contain navigation bar at the correct position in top navBarLocation mode', () => {
-    wizardTest.wizard.navBarLocation = 'top';
+    wizard.navBarLocation = 'top';
     wizardTestFixture.detectChanges();
 
     const navBar = wizardTestFixture.debugElement.query(By.css('aw-wizard-navigation-bar'));
-    const wizard = wizardTestFixture.debugElement.query(By.css('aw-wizard'));
+    const wizardEl = wizardTestFixture.debugElement.query(By.css('aw-wizard'));
```

Comment:
If we rename a debug element according to a new naming scheme (i.e. appending `El`) because of a naming conflict I think we should rename all debug elements to retain consistency

Reply:
Done in 536b136.